File: practice_4-23/Prac.java
public class Prac {
    public static void main(String[] args) {
        int[] intArray = { 1, 1, 0, 0, 0 };
        System.out.println(computeArray(intArray));
    }

    public static boolean computeArray(int[] intArray) {
        int zeros = 0;
        int ones = 0;
        for (int i : intArray) {
            if (i == 1) {
                ones++;
            } else if (i == 0) {
                zeros++;
            }
        }
        // int i = 0;
        // while (i < intArray.length) {
        // if (intArray[i] == 1) {
        // ones++;
        // } else if (intArray[i] == 0) {
        // zeros++;
        // }
        // i++;
        // }
        return ones > zeros;
    }
}

File: practice_4-23/prac.py
def compute_list(int_list):
    ones = 0
    zeros = 0
    for i in int_list:
        if (i == 1):
            ones += 1
        elif (i == 0):
            zeros += 1
    # i = 0
    # while i < len(int_list):
    #     if int_list[i] == 1:
    #         ones += 1
    #     elif int_list[i] == 0:
    #         zeros += 1
    #     i += 1
    print(ones > zeros)
    return ones > zeros


def main():
    int_list = [0, 1, 1, 4, 7]
    compute_list(int_list)


if __name__ == "__main__":
    main()
